packages/core/certora/README.md

```diff
@@ -0,0 +1,41 @@
+## Verification Overview
```

Comment:
as a general comment on all the stuff in the `certora` directory: you are only testing _one_ contract here: the `optimistic-asserter`. it might be useful to move pretty much everything you have in `/core/certora` to `/core/cetora/optimistic-asserter` so that we can expand for other contract types, grouped by categories. 

basically, I want us to be able to expand the number of these tests by having additional directories in the same way we have directories for each contract type within the `core/contracts` dir.

let me know what you think on this!

Reply:
The arranging of directories is completely up to you. Given that there would be more than one contract to be verified, what you're suggesting totally makes sense.
I would re-arrange the certora dir and adapt the script.